qurator/dinglehopper/extracted_text.py

```diff
@@ -246,6 +243,7 @@ def get_textequiv_unicode(text_segment, nsmap) -> str:
 
 def get_first_textequiv(textequivs, segment_id):
     """Get the first TextEquiv based on index or conf order if index is not present."""
+    LOG = getLogger('processor.OcrdDinglehopperEvaluate')
```

Comment:
```suggestion
    log = getLogger('processor.OcrdDinglehopperEvaluate')
```

Reply:
Then you'll have to rename the usage, too.